docs/src/gallery/geometries.md

```diff
@@ -0,0 +1,81 @@
+# Geometries
+
+## Geom.point
+
+```@example
+using Gadfly, RDatasets
+Gadfly.set_default_plot_size(21cm, 12cm) # hide
+
+D = dataset("datasets", "iris")
+p1 = plot(D, x="SepalLength", y="SepalWidth", Geom.point);
+
+# Binding continuous data to the color aesthetic
+p2 = plot(D, x="SepalLength", y="SepalWidth", color="PetalLength", Geom.point);
+
+# Binding categorial data to the color aesthetic
+p3 = plot(D, x="SepalLength", y="SepalWidth", color="Species", Geom.point);
+
+# Binding categorical data to the shape aesthetic
+p4 = plot(D, x="SepalLength", y="SepalWidth", color="Species", shape="Species",
+          Geom.point);
+
+gridstack([p1 p2; p3 p4])
+```
+
+
+```@example
+# Binding categorial data to x
+using Gadfly, RDatasets
+Gadfly.set_default_plot_size(14cm, 8cm) # hide
+plot(dataset("lattice", "singer"), x="VoicePart", y="Height", Geom.point)
+```
+
+```@example
+# Different colored layers
+using Gadfly, Distributions
+Gadfly.set_default_plot_size(14cm, 8cm) # hide
+rdata = rand(MvNormal([0,0.],[1 0;0 1.]),100)
+bdata = rand(MvNormal([1,0.],[1 0;0 1.]),100)
+plot(layer(x=rdata[1,:], y=rdata[2,:], color=[colorant"red"], Geom.point),
+     layer(x=bdata[1,:], y=bdata[2,:], color=[colorant"blue"], Geom.point))
+```
+
+## Geom.bar
+
+```@example
+using Gadfly, RDatasets
+set_default_plot_size(14cm, 8cm) # hide
+plot(dataset("HistData", "ChestSizes"), x="Chest", y="Count", Geom.bar)
+```
+
+```@example
+using Gadfly, RDatasets, DataFrames
+set_default_plot_size(21cm, 8cm) # hide
+
+D = by(dataset("datasets","HairEyeColor"), [:Eye,:Sex], d->sum(d[:Freq]))
+p1 = plot(D, color="Eye", y="x1", x="Sex", Geom.bar(position=:dodge),
+          Guide.ylabel("Freq"));
+
+rename!(D, :x1 => :Frequency)
+palette = ["blue","brown","green","tan"]  # Is there a hazel color?
```

Comment:
Leftover comment? This is unnecessary

Reply:
Not leftover, it's a humorous comment!  Hazel is a color (think eyes/hazelnut),  and the plot is about eye color, so one should be able to use "hazel"!

Reply:
Ah. I get it.